stories/multi-combo-box/multi-combo-box.stories.js

```diff
@@ -0,0 +1,960 @@
+The \`popover\` is shared between the combobox and select components. Please see the \`select\` documentation for the complete list of modifiers,
+That are also  supported by the \`combobox\` component.
+
+\`Multi ComboBox\` allows users to enter multiple values which are displayed as a tokens. 
+It provides an editable input field for filtering the list, and a dropdown menu with a list of the available options. 
+If the entries are not validated by the application, users can also enter custom values. 
+`,
+        tags: ['f3', 'a11y', 'theme'],
+        components: [
+            'form-label',
+            'popover',
+            'radio',
+            'title',
+            'token',
+            'input-group',
+            'checkbox',
+            'list',
+            'button',
+            'icon',
+            'tokenizer',
+            'dialog',
+            'bar',
+            'input'
+        ]
+    }
+};
+
+export const cozyAndCompact = () => `<div style="display:flex;height:310px">
+    <div style="display:flex;flex-direction:column;padding-right:1rem">
+        <label class="fd-form-label">
+            Multi ComboBox Cozy Mode
+        </label>
+        <div class="fd-popover">
+            <div class="fd-popover__control" aria-controls="F4GcX348a" aria-expanded="true" aria-haspopup="true">
+                <div class="fd-input-group fd-input-group--control">
+                    <div class="fd-tokenizer">
+                        <div class="fd-tokenizer__inner">
+                            <span class="fd-token" role="button">
+                                <span class="fd-token__text">
+                                    Apple
+                                </span>
+                                <button class="fd-token__close"></button>
+                            </span>
+                            <span class="fd-token" role="button">
+                                <span class="fd-token__text">
+                                    Orange
+                                </span>
+                                <button class="fd-token__close"></button>
+                            </span>
+                            <span class="fd-token" role="button">
+                                <span class="fd-token__text">
+                                    Kiwi
+                                </span>
+                                <button class="fd-token__close"></button>
+                            </span>
+                            <span class="fd-token" role="button">
+                                <span class="fd-token__text">
+                                    Banana
+                                </span>
+                                <button class="fd-token__close"></button>
+                            </span>
+                            <input class="fd-input fd-input-group__input fd-tokenizer__input" />
```

Comment:
These should be `role="combobox"` with the associated attributes.
```suggestion
                            <input class="fd-input fd-input-group__input fd-tokenizer__input" role="combobox" aria-expanded="true" aria-haspopup="listbox" />
```